test/supertest.js

```diff
@@ -150,7 +154,7 @@ describe('request(app)', function(){
 
     request(app)
     .get('/')
-    .expect(302, done);
+    .expect(302, function() { done(); });
```

Comment:
FYI this test is now useless; try changing the `302` to `400` and you'll see it still passes. This is because you ignore all errors, even if the error is valid, meaning the test doesn't test anything at all.